test/transformers/map.js

```diff
@@ -66,13 +67,9 @@ describe('map', function() {
         })
 
         it('Should parse an array correctly', function() {
-            const callback = function(p) {
-                assert(p instanceof Parse);
-                assert(p._chain[0]);
-                assert.notEqual(typeof p._chain[0]._path, 'undefined');
-
+            const callback = function() {
```

Comment:
Could you put this assert back in? It tests that `p` is correctly set to a Parse instance.
```suggestion
            const callback = function(p) {
                assert(p instanceof Parse);
```